soccer-records-model/src/test/java/cz/muni/fi/pa165/team/TeamMatchGoalRepositoryImplTest.java

```diff
@@ -171,22 +173,9 @@ public void testFindMatchByScorer()
         em.persist(goal1);
         em.persist(goal2);
         em.persist(goal3);
-
-        Collection<TeamMatchGoal> dbGoals = teamMatchGoalDAO.findGoalByScorer(scorer);
-
         em.flush();
 
-        Assert.assertEquals(dbGoals.size(), 2);
-        Assert.assertTrue(dbGoals.contains(goal1));
-        Assert.assertTrue(dbGoals.contains(goal3));
-        Assert.assertFalse(dbGoals.contains(goal2));
-    }
-
-    @Test(expectedExceptions = IllegalArgumentException.class)
-    public void testFindGoalByScorerNull()
-    {
-        Collection<TeamMatchGoal> dbGoals = teamMatchGoalDAO.findGoalByScorer(null);
-        em.flush();
+        Collection<TeamMatchGoal> dbGoals = teamMatchGoalRepository.findGoalByScorer(null);
```

Comment:
* je nutny si tu připravovat data, když testuješ vyjimku, ktera tu metodu hned z kraje ukončí? `IllegalArgumentException`
* je nutny to ukladat do lokalni promenne, kdyz ji na nic nepotřebuješ?